guides/06_Meeting_Events.md

```diff
@@ -204,6 +206,8 @@ The following table lists available states.
 |`videoInputFailed`|The camera selection failed.
 |`videoInputSelected`|The camera was selected.
 |`videoInputUnselected`|The camera was removed. You called `meetingSession.audioVideo.stopVideoInput` with `null`.
+|`sendingAudioFailed`|Audio packets are not being sent out for a considerable timespan
```

Comment:
The state description here for both the new events is different from the ones above under "Meeting events and attributes", can you please keep them same?